WordPress/Classes/ViewRelated/Post/AbstractPostListViewController.swift

```diff
@@ -805,8 +805,8 @@ class AbstractPostListViewController: UIViewController, WPContentSyncHelperDeleg
         WPAnalytics.track(.postListPublishAction, withProperties: propertiesForAnalytics())
 
         apost.status = PostStatusPublish
-        if let date = apost.dateCreated(), date == (Date() as NSDate).laterDate(date) {
-            apost.setDateCreated(Date())
+        if let date = apost.dateCreated, date == (Date() as NSDate).laterDate(date) {
```

Comment:
Can you add a comment here to explain this if, was having and hard time to understand what you where doing here.